cms/static/js/views/modals/move_xblock_modal.js

```diff
@@ -88,6 +98,67 @@ function($, Backbone, _, gettext, BaseView, BaseModal, XBlockInfoModel, MoveXBlo
                     model: new XBlockInfoModel(outlineJson, {parse: true})
                 }
             );
+        },
+
+        showMovedBar: function(title, titleLink, messageLink) {
+            var self = this;
+            if (self.movedAlertView) {
+                self.movedAlertView.hide();
+            }
+            self.movedAlertView = new MovedAlertView({
+                title: title,
+                titleLink: titleLink,
+                messageLink: messageLink,
+                maxShown: 10000
+            });
+            self.movedAlertView.show();
+            // scroll to top
+            $.smoothScroll({
+                offset: 0,
+                easing: 'swing',
+                speed: 1000
+            });
+        },
+
+        moveXBlock: function() {
+            var self = this;
+            XBlockViewUtils.moveXBlock(self.sourceXBlockInfo.id, self.moveXBlockListView.parent_info.parent.id)
```

Comment:
Not sure if it's correct way to pick up parent id `self.moveXBlockListView.parent_info.parent.id`